realm-transformer/build.gradle

```diff
@@ -30,7 +30,6 @@ dependencies {
     compile gradleApi()
     compile "io.realm:realm-annotations:${version}"
     compile 'com.android.tools.build:gradle:1.5.0'
```

Comment:
Should this be provided instead? 

Build tools version might matter quite a lot to people so if we force some minimum version onto them, other subtle things might break.

I would probably be more comfortable with A) having this as provided B) Document that 1.5 is the minimum version we support and C) Just let user projects crash if they use below 1.5

What do you think?